devices.go

```diff
@@ -31,6 +31,7 @@ type Device struct {
 	Updated      string       `json:"updated_at,omitempty"`
 	Locked       bool         `json:"locked,omitempty"`
 	BillingCycle string       `json:"billing_cycle,omitempty"`
+	IPXEUrl      string       `json:"ipxe_script_url,omitempty"`
```

Comment:
Let's use `IPXEScriptUrl` to keep it consistent with the API parameter / property.